composer.json

```diff
@@ -19,6 +19,12 @@
             "homepage": "https://github.com/Sylius/Sylius/contributors"
         }
     ],
+    "repositories": [
+        {
+            "type": "vcs",
+            "url": "https://github.com/AdamKasp/Calendar"
+        }
+    ],
```

Comment:
`sylius/calendar` repository is [already on Packagist](https://packagist.org/packages/sylius/calendar), so we should be able to use it without using a custom repository setting. What we definitely should do is release a 0.1 version of it to not use `dev-master` branch 🖖